docs/src/guide/installation.md

```diff
@@ -0,0 +1,43 @@
+# Installation
+
+DFTK is not yet registered in the [General](https://github.com/JuliaRegistries/General)
+registry of Julia.
+Instead you can obtain it from
+the [MolSim](https://github.com/JuliaMolSim/MolSim.git) registry,
+which contains a bunch of packages related to performing molecular simulations in Julia.
+At least **Julia 1.3** is required.
+
+First add `MolSim` to your installed registries. For this use
+```
+] registry add https://github.com/JuliaMolSim/MolSim.git
+```
+from a Julia command line.
```

Comment:
This is maybe not necessary... I'd just tell people to add the DFTK github

Reply:
Let's not discuss this. Will be gone in a week anyway.